ynput/OpenPype
# Deadline: Remove redundant instance_skeleton_data code - OP-8269

## Changelog Description
This PR https://github.com/ynput/OpenPype/pull/5186 re-introduced code about for the `instance_skeleton_data` but its actually not used since this variable gets overwritten later.

## Testing notes:
1. Submit to Deadline from all hosts should work.
2. Publish on Deadline from all hosts should work.
